modules/fxgui/src/edu/kit/iti/algover/rule/script/ScriptController.java

```diff
@@ -271,7 +368,7 @@ public void onAsyncScriptTextChanged(String text) {
         //neuberechnen -> User
         //onCaretPositionChanged(null);
 
-          view.setStyle("-fx-background-color: #c4c1c9;"+fontSizeProperty.get()+"pt;");
+          view.setStyle("-fx-background-color: lightgray; -fx-font-size: "+fontSizeProperty.get()+"pt;");
```

Comment:
this should also disable the changing of sequents since we cannot reliably show the right sequent until runScript is executed again